.github/secrel/config.yml

```diff
@@ -10,21 +10,44 @@
 #   - -Dexample-build-arg=some-value
 #   - -Dexample.maven=${{ secrets.example_secret }}
 
+snyk:
+  additional-args:
+  - -PGITHUB_ACCESS_TOKEN=${{ secrets.ACCESS_TOKEN }}
+
 # Remove image block if you're not building Docker images
 images:
-- name: app-image
-  path: "./Dockerfile"
-  args:
-  - example-build-arg=some-value
-  - example-build-arg-secret=${{ secrets.example_secret }}
+  - name: abd_vro
+    path: "./Dockerfile"
+    args:
+      -  GITHUB_ACCESS_TOKEN=${{ secrets.ACCESS_TOKEN }}
+  - name: abd_vro-db-init
+    path: "./db-init/src/docker/Dockerfile"
```

Comment:
When this change was made in LH StarterKit: https://github.com/department-of-veterans-affairs/lighthouse-di-demo-java-starterkit/commit/eb6d84fd3cdc02ab2ac97875af086598b53514d3#diff-fd8faa09e718dcbb6de1bf4534d2c8aaac1c439366bbeeaaea6aef60e4dd474a